pkg/linter/linter.go

```diff
@@ -111,8 +111,12 @@ type Location struct {
 }
 
 func (l Location) String() string {
-	return fmt.Sprintf("%s: %s step in the %s mixin (%s)",
-		l.Action, humanize.Ordinal(l.StepNumber), l.Mixin, l.StepDescription)
+	mixin := ""
```

Comment:
Looking at the original issue, I think the logic should be something along the lines of if the mixin name is "", we should not be returning a string. Let me know if you are reading that from #2526

Reply:
You are complety correct, I will work on the change

Reply:
@schristoff It has been changed now to align with your comment